### ISSUE TITLE
Improve example line chart

### ISSUE THREAD

**Issue by cjrace** [dfe-analytical-services/shiny-template#99]
Line chart should be made in ggplot2 / afcharts with ggiraph for the tooltips / interactivity.

Can take a look at the [LAIT](https://department-for-education.shinyapps.io/local-authority-interactive-tool/?_inputs_&year_range-year_range=null&navsetpillslist=%22LA%20Level%22&la_inputs-la_name=%22Barking%20and%20Dagenham%22&la_inputs-indicator_name=%22A%20level%20cohort%20Average%20point%20score%20per%20entry%22&region_inputs-la_name=%22Barking%20and%20Dagenham%22&region_inputs-indicator_name=%22A%20level%20cohort%20Average%20point%20score%20per%20entry%22&stat_n_inputs-la_name=%22Barking%20and%20Dagenham%22&stat_n_inputs-indicator_name=%22A%20level%20cohort%20Average%20point%20score%20per%20entry%22&all_la_inputs-la_name=%22Barking%20and%20Dagenham%22&all_la_inputs-indicator_name=%22A%20level%20cohort%20Average%20point%20score%20per%20entry%22&create_inputs-geog_input=null&create_inputs-indicator=null&create_inputs-la_group=%22no_groups%22&create_inputs-inc_regions=false&create_inputs-inc_england=false) for examples.

We should update the chart so that it includes a table behind it as well as download options. All of it should sit within a bslib card component.

We should also include an option to copy image of the chart to clipboard.

We should double check that the chart is following [best practice guidelines from the AF guidance hub](https://analysisfunction.civilservice.gov.uk/policy-store/data-visualisation-charts/), using afcolours / afcharts packages and also labelling lines directly.
